src/core/pages/linkUtils.ts

```
@@ -0,0 +1,45 @@
+  originalUrl: string;
+  params: MediaParams | null;
+} {
+  const parts = url.split('::');
+  if (parts.length !== 2) return { originalUrl: url, params: null };
+
+  const [width, height, fullscreen] = parts[1].split(',');
+
+  return {
+    originalUrl: parts[0],
+    params: {
+      ...(width && { width: parseInt(width) }),
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)